client/signup/steps/website-content/wordpress-media-upload.tsx

```diff
@@ -0,0 +1,116 @@
+import styled from '@emotion/styled';
+import { useState } from 'react';
+import placeholder from 'calypso/assets/images/difm/placeholder.svg';
+import FilePicker from 'calypso/components/file-picker';
+import Spinner from 'calypso/components/spinner';
+import { useAddMedia } from 'calypso/data/media/use-add-media';
+import { SiteData } from 'calypso/state/ui/selectors/site-data';
+import { Label, SubLabel } from './components';
+
+const UPLOAD_STATES = {
+	NOT_SELECTED: 'NOT_SELECTED',
+	IN_PROGRESS: 'IN_PROGRESS',
+	COMPLETED: 'COMPLETED',
+	FAILED: 'FAILED',
+};
+
+const FileSelectThumbnailContainer = styled.div`
+	min-width: 190px;
+	width: 190px;
+	max-height: 145px;
+	background: rgba( 187, 224, 250, 0.12 );
+	border: 1px dashed #dcdcde;
```

Comment:
We can use CSS variables here.
```suggestion
	border: 1px dashed var( --studio-gray-5 );
```